src/components/templates/PosterHall/components/PosterPreview/PosterPreview.scss

```diff
@@ -56,4 +56,15 @@ $active-poster-shadows: 0 10px 20px 0 rgba($primary, 0.56);
     font-size: $font-size--sm;
     font-weight: $font-weight--500;
   }
+
+  &__bookmark {
+    cursor: pointer;
+    height: 100%;
+    display: flex;
+    justify-content: right;
+    align-items: right;
+    top: 0%;
+    padding: 0.5rem;
+    color: rgb(255, 255, 255);
+  }
```

Comment:
Where these styles written from scratch, or copied from somewhere? They seem to differ from `&__bookmark` in `ScheduleEvent.scss`. Why is that?

- https://github.com/sparkletown/sparkle/blob/staging/src/components/molecules/ScheduleEvent/ScheduleEvent.scss#L88-L98

Reply:
The bookmark on the poster preview cards needs to be positioned differently than the schedule event rectangle because of the layout differences. For the most part, the css is similar, but the additions are generally to address the positioning. One other addition is the `cursor: pointer`, because it helps to indicate the unique function associated with clicking the bookmark icon to the user

Reply:
That makes sense. Sounds good.

Does the bookmark not have this cursor on the schedule? If not, would be good to add it there as well.

Reply:
done